### PR TITLE
Updated to use a standalone version of a Messaging Service

### PR DESCRIPTION
**Pull request by kirlat**
This build switches to use an `alpheios-messaging` package instead of `lexis-cs`.

I will remove branch dependencies after the messaging service changes are merged.

### CODE REVIEW

**Review by balmas — APPROVED** [review]
thanks

File: package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "alpheios-experience": "github:alpheios-project/experience",
     "alpheios-fixtures": "github:alpheios-project/fixtures",
     "alpheios-inflection-tables": "github:alpheios-project/inflection-tables",
-    "alpheios-lexis-cs": "github:alpheios-project/lexis-cs",
+    "alpheios-messaging": "github:alpheios-project/alpheios-messaging#mss-service-split",
```
Comment (irina060981) [inline package.json]: I believe the branch would be removed before merging.

**Review by irina060981 — APPROVED** [review]
Looks ok!

File: package.json (hunk 1 above)
Comment (kirlat) [reply package.json]: Yes, I will remove it

**Pull request merged**